app/views/user_mailer/new_auth0_user_new_project.html.erb

```diff
@@ -0,0 +1,8 @@
+<p><b>Join IDseq!</b></p>
```

Comment:
This is copied with minor variable name changes from `app/views/devise/mailer/new_user_new_project.html.erb`, which should be deleted later